executor/aggfuncs/aggfunc_test.go

```diff
@@ -137,7 +137,7 @@ func (p *multiArgsAggTest) genSrcChk() *chunk.Chunk {
 			srcChk.AppendDatum(j, &fdt)
 		}
 	}
-	srcChk.AppendDatum(0, &types.Datum{})
+	//srcChk.AppendDatum(0, &types.Datum{})
```

Comment:
why do we remove this line?

Reply:
Because this empty line will cause JSON parsing to fail. I think it is possible to add a method here instead of removing it. In addition, we should find out why others add this line here.

Reply:
I think it is ok to have an error here, wrong input is also a test case. We can check whether `datum` is null in `defaultMultiArgsMemDeltaGens` to handle this case.